src/Settings/Objects.ts

```diff
@@ -580,6 +580,13 @@ export function GenerateObjectsSettings(): GameStartr.IObjectMakrCustoms {
                     "HPBar": {},
                     "Note": {},
                     "ExplosionSmall": {},
+                    "IceBeamCircle": {},
+                    "IceBeamCircleReverse": {},
+                    "IceBeamLine": {},
+                    "IceBeamIcicles1": {},
+                    "IceBeamIcicles2": {},
+                    "IceBeamIcicles3": {},
+                    "IceBeamIcicles4": {},
```

Comment:
Name these something better, perhaps? Something to do with partial/half/sections/etc? One should be able to intelligently guess what they are based on the name.